wpsc-admin/admin.php

```diff
@@ -1474,6 +1477,46 @@ function wpsc_duplicate_product_meta( $id, $new_id ) {
 }
 
 /**
+ * Duplicate Featured Image
+ *
+ * When a product is duplicated, the featured image ID is copied when the post
+ * meta is duplicated.
+ * 
+ * When the featured image is attached to the duplicated product, if the image
+ * is duplicated the featured image ID is updated to the duplicated image ID
+ * otherwise the featured image ID is removed.
+ *
+ * If the featured image is not attached to the product the featured image ID
+ * remains the same as the original product.
+ *
+ * This function will remove the featured image if the image is not attached to
+ * the duplicated product and offers the opportunity to change the featured image
+ * of the duplicated product via the 'wpsc_duplicate_product_thumbnail' filter.
+ *
+ * @param   integer  $id      Product ID.
+ * @param   integer  $new_id  Duplicated product ID.
```

Comment:
These need to be updated to reflect the updated parameter names (`$post_id` and `$new_post_id`)